src/api/app/models/user_ldap_strategy.rb

```diff
@@ -361,7 +361,7 @@ def relationship_groups_contains_user?(relationships, method_name)
     relationships.each do |relationship|
       return false if relationship.group.nil?
       # check whether current user is in this group
-      return true if user_in_group_ldap?(login, relationship.group)
+      return true if user_in_group_ldap?(@login, relationship.group)
```

Comment:
I don't think the instance variable is the problem here. I don't see it being initialized anywhere. `login` for sure wouldn't work either. I think what you want here is `User.session`

Reply:
As said, I have zero Ruby knowledge. With those changes, though, LDAP groups work for me - provided that I create them in OBS first, with the proper name. Not sure if this is intended behavior. Also, no members are listed in the "/groups" Web-UI page, but the membership checks seem to work correctly.

The feature certainly does not seem widely used, if at all ;)